Source/DafnyCore/Backends/CSharp/CsharpCodeGenerator.cs

```diff
@@ -67,7 +67,7 @@ protected override void EmitHeader(Program program, ConcreteSyntaxTree wr) {
       wr.WriteLine("// Dafny program {0} compiled into C#", program.Name);
       wr.WriteLine("// To recompile, you will need the libraries");
       wr.WriteLine("//     System.Runtime.Numerics.dll System.Collections.Immutable.dll");
-      wr.WriteLine("// but the 'dotnet' tool in net6.0 should pick those up automatically.");
+      wr.WriteLine("// but the 'dotnet' tool in net8.0 should pick those up automatically.");
```

Comment:
```suggestion
      wr.WriteLine("// but the 'dotnet' tool in .NET should pick those up automatically.");
```

Just to avoid some diffs the next time we upgrade.